internal/pkg/deploy/cloudformation/cloudformation.go

```diff
@@ -427,3 +428,30 @@ func stopSpinner(spinner *progress.Spinner, err error, label string) {
 	}
 	spinner.Stop(log.Serrorf("%s\n", label))
 }
+
+// upgradeStackUpdateErrorHandling is used to handle the error returned by CFN
+// stack update when upgrading the application/environment.
+func upgradeStackUpdateErrorHandling(name string, err error) (bool, error) {
```

Comment:
Can we add one sentence in the function-level comment that explains the return value (especially the `bool`)? Although this can be inferred from the code comments, an explicitly description in function-level comment will still be helpful.